src/test/features/ccj/routes/repayment-plan-summary.ts

```diff
@@ -15,10 +15,11 @@ import { checkAuthorizationGuards } from 'test/features/ccj/routes/checks/author
 import { checkNotClaimantInCaseGuard } from 'test/features/ccj/routes/checks/not-claimant-in-case-check'
 import { MomentFactory } from 'shared/momentFactory'
 import { CountyCourtJudgmentType } from 'claims/models/countyCourtJudgmentType'
+import { MadeBy } from 'offer/form/models/madeBy'
 
 const cookieName: string = config.get<string>('session.cookieName')
 const externalId = claimStoreServiceMock.sampleClaimObj.externalId
-const pagePath = Paths.repaymentPlanSummaryPage.evaluateUri({ externalId: externalId })
+const pagePath = Paths.repaymentPlanSummaryPage.evaluateUri({ externalId: externalId, madeBy: MadeBy.CLAIMANT.value })
```

Comment:
same as above. 
https://github.com/hmcts/cmc-citizen-frontend/pull/1143/files#r232730772

Reply:
RoutablePath has test for more than one  evaluation in uri as well as null, undefined\